### ISSUE TITLE
Not inject on latest Canary build

### ISSUE THREAD

**Issue by XCanG** [leovoel/BeautifulDiscord#51]
`app-0.0.204`

`[BUILD INFO] Release Channel: canary, Build Number: 14306, Version Hash: c6d6ff0164901ec2bdce707e57891c2a569b146a`

**Comment by shikhir-arora** [comment]
I don't have an issue, same build. You sure you running the latest master branch?

**Comment by XCanG** [comment]
Yes, it update frequently. I'll tell if it will not work on next update.

**Comment by XCanG** [comment]
I recieve new update and B.D. still not work for me.

`[BUILD INFO] Release Channel: canary, Build Number: 14388, Version Hash: 774c099644f872bb98557e9ae1ee936fe14479aa`

**Comment by shikhir-arora** [comment]
I can't reproduce this still. Same build as you.

Yes, after a 'hard' update on Canary (where you have to restart the client) - I do have to re-run `beautifuldiscord --css` 99%+ of the times, but that is expected and takes like 2 seconds to do (since my CSS file doesn't change in itself as I just use imports for everything). I use macOS/Linux so I can only speak to that. 

Are you *sure* you are running the latest branch (`master`, not `canary`) of BeautifulDiscord itself? The canary branch doesn't work/hasn't been updated for a while, but no need as the master branch works properly. Does it find the executable and try to inject? Do you have a screenshot if so?

The only thing that I can think of is if you are using Windows, on stable maybe the recent updates last Friday for voice/overlay (but only for the stable build) might have broke it. However, I seriously doubt that is the case since other Windows users seemingly can't reproduce this either, and it was just pertaining to the stable client. 

Can see the screenshots below in my case, after updating Discord just now:

---

![screen shot 2018-05-01 at 5 26 08 am](https://user-images.githubusercontent.com/8797918/39468518-5e01d380-4d01-11e8-9d30-868817de5e7d.png)
![screen shot 2018-05-01 at 5 30 40 am](https://user-images.githubusercontent.com/8797918/39468520-606d7e8a-4d01-11e8-8628-e11b08928a3e.png)

**Comment by XCanG** [comment]
With latest update (`app-0.0.206`, `[BUILD INFO] Release Channel: canary, Build Number: 14432, Version Hash: b1e0325580b21e818cad6c7aae22115a1ca8b7d8`) I got it to work, but I get new error, may be it not related to B.D., but well:
```
C:\Users\X4\AppData\Local\DiscordCanary\app-0.0.206\resources\electron.asar\renderer\init.js:162
Electron Security Warning This renderer process has Node.js integration enabled and attempted to load remote content. This exposes users of this app to severe security risks.
For more information and help, consult https://electronjs.org/docs/tutorial/security
```
Just to mention:
```
EvilDabbitBOT - Today at 04:33
@Canary A new canary build is coming for all desktop platforms in ~10m. The only substantial change is an upgrade to Electron. Please report all the spoopy issues.
```

**Comment by XCanG** [comment]
Wait, it work only partially, hot reload not work now (when I make changes to fie and save), styles now updated only if I restart discord.

**Comment by Wittyy** [comment]
@shikhir-arora Is that your own theme? If so, mind sharing? Really love the way it looks.

**Comment by shikhir-arora** [comment]
@AlexFlipnote Thanks for the awesome theme and plugins! And yes, just reloaded with latest Canary and the above mentioned CSS file with the plugins and it works just fine. Would expect nothing less, as the updates were mainly for Electron, so it shouldn't change the CSS injection/node properties.

**Comment by XCanG** [comment]
Just a reminder what with latest build it get work for me but without hot reload.
